reference-implementation/to-upstream-wpts/writable-streams/close.js

```diff
@@ -213,4 +219,96 @@ promise_test(() => {
   });
 }, 'the promise returned by async abort during close should resolve');
 
+// Though the order of the promises is not important, we're checking it for interoperability. We can change the order
```

Comment:
I suggest "arbitrary" rather than "not important". Also, maybe "order in which the promises are fulfilled or rejected"?